R/plumber.R

```diff
@@ -182,12 +182,14 @@ plumber <- R6Class(
 
     },
     run = function(host='127.0.0.1', port=getOption('plumber.port'), swagger=interactive(),
-                   debug=interactive()){
+                   debug=interactive() ){
       port <- findPort(port)
 
       message("Starting server to listen on port ", port)
 
-      private$errorHandler <- defaultErrorHandler(debug)
+      on.exit({ options('plumber.debug' = getOption('plumber.debug')) })
```

Comment:
Shoot. Sorry. Last thing. I just noticed this is going to get clobbered by the `on.exit()` call on line 197. Can you throw an `add=TRUE` onto 197?